bedrock/events/models.py

```diff
@@ -14,40 +14,58 @@
 from bedrock.events.countries import country_to_continent
 
 
+def calendar_id_from_google_url(feed_url):
+    return feed_url.split('/')[5]
+
+
+def calendar_url_for_event(event, calendar_id):
+    template = 'https://www.google.com/calendar/embed?showCalendars=0&mode=week&wkst=1&' \
```

Comment:
That'd be fine with me. I just thought that the agenda view looked weird as a full page. We could popup a window, but that's pretty lame. I'm fine with whatever people want though really.